File: C#/Polymorphism/Polimorfizm/Zwierze.cs
namespace Polimorfizm;

public class Zwierze
{
    private int a = 1; 
    internal int b = 2;
    public int c = 3;
    protected int d = 4;
    
    
    virtual public void DajGlos()
    {
        Console.WriteLine("Głos zwierzęcia - nie wiem bo nie wiem jakim jestem zwierzęciem");
    }
    
    virtual public void Zjedz()
    {
        Console.WriteLine("Pokarm - nie wiem bo nie wiem jakim jestem zwierzęciem");
    }

    public Zwierze()
    {
        Console.WriteLine("Jestem zwierzęciem");
    }

    
}

File: Java/Nonmobile/Polymorphism/Zwierze.java
package com.example.javalib;

public class Zwierze {

    int waga;
    public Zwierze(int waga){
        this.waga = waga;
        System.out.println("Jestem zwierzęciem");
    }

    public Zwierze(){
        System.out.println("Jestem zwierzęciem");
    }
    public void oddycham(){
        System.out.println("oddycham");
    }
    public void dajGlos(){
        System.out.println("?");
    }

    public void czymSieKarmie(){
        System.out.println("Karmię się: ?");
    }

}
